apps/src/gamelab/animationListModule.js

```diff
@@ -231,11 +231,11 @@ function generateAnimationName(baseName, animationList) {
   // Match names with the form baseName_#
   for (let animation in animationList) {
     let animationName = animationList[animation].name;
-    if (animationName.startsWith(baseName)) {
+    if (animationName.lastIndexOf(baseName, 0) === 0) {
```

Comment:
Any reason this can't just be `indexOf`?

Could also do `animationName.substring(0, baseName.length) === baseName` I think.